recipes/xerces-c28/build.sh

```diff
@@ -0,0 +1,39 @@
+#! /bin/bash
+
+set -e
+
+if [ -n "$OSX_ARCH" ] ; then
+    export MACOSX_DEPLOYMENT_TARGET=10.7 # C++ libc++ needs this
```

Comment:
Yeah, I added the SDK flags in my personal build scripts as future-proofing. My understanding is that MacOS SDKs are good with backwards-compat so that DEPLOYMENT_TARGET should be all you need to set, but it didn't seem like it hurt to have the flags.

Looks like `toolchain` targets a minimum of 10.9? I just chose my setting as the oldest that would work. I presume the `toolchain` setting was chosen carefully so I'll switch the package to not override it.